charts/internal/cilium/charts/operator/templates/clusterrole.yaml

```diff
@@ -151,6 +151,13 @@ rules:
   - watch
   - delete
   - patch
+- apiGroups:
```

Comment:
Why are `secrets` missing in the section `# to check apiserver connectivity`? If I diff the versions, it seems like this should have been added.

Reply:
`secrets` would be needed if `.Values.tls.secretSync.enabled ` would be true.
That would be part of TLS interception part, we want to add later.